src/referencePackages/src/system.runtime.interopservices/4.3.0/System.Runtime.InteropServices.4.3.0.csproj

```diff
@@ -43,9 +40,6 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
-    <!-- Needs additional APIs that don't exist in the contract. -->
```

Comment:
You are reverting some intentional tweaks to the generated code.  The changes to this file as well as the deletion of https://github.com/dotnet/source-build-reference-packages/pull/1139/files#diff-217e4e4d9fa100fb4df1c2ff5c488dc55e6e4c9c6fe15f4c9b4c7bce389bac72 will need to be reverted.

Reply:
reverted.